plugins/ExamplePlugin/src/ProjectileCodePlugin.py

```diff
@@ -0,0 +1,131 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Created on Oct. 28, 2022
+
+@author: alfoa, wangc
+
+comments: Interface for Projectile Code through code plugin
+"""
```

Comment:
I can add the note in the code. Ideally I would like to avoid duplications, however, I also want the code plugin test to be independent. If you have better idea, please let me know.

Reply:
Yes, adding (or adding to) a comment is fine.

Reply:
added.